app/coffeescripts/views/conferences/ConferenceView.coffee

```diff
@@ -126,3 +128,81 @@ define [
         else
           window.open(data[0].url)
       )
+
+    deleteRecording: (e) ->
+      return if !confirm(I18n.t('recordings.confirm.delete', "Are you sure you want to delete this recording?"))
```

Comment:
We don't use keys anymore with I18n so we can remove `'recordings.confirm.delete',` here.